src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.Application.targets

```diff
@@ -27,8 +25,29 @@ This file contains targets specific for Android application projects.
       Install;
       StartAndroidActivity;
     </_RunDependsOn>
+    <_AndroidComputeRunArgumentsDependsOn>
+      Install;
+    </_AndroidComputeRunArgumentsDependsOn>
   </PropertyGroup>
 
+  <Target Name="_AndroidComputeRunArguments"
+      BeforeTargets="ComputeRunArguments"
+      DependsOnTargets="$(_AndroidComputeRunArgumentsDependsOn)">
+    <GetAndroidActivityName
+        Condition=" '$(AndroidLaunchActivity)' == '' "
+        ManifestFile="$(IntermediateOutputPath)android\AndroidManifest.xml">
+      <Output TaskParameter="ActivityName" PropertyName="AndroidLaunchActivity" />
+    </GetAndroidActivityName>
+    <PropertyGroup>
+      <RunCommand>$(AdbToolExe)</RunCommand>
```

Comment:
Elsewhere, we support `$(AdbToolPath)` + `$(AdbToolExe)`.  As-is, it looks like this requires that `$(AdbToolExe)`/`adb` be in `%PATH%`

Reply:
The current code doesn't require `adb` to be in `$PATH`. It's working on my machine without it, for example:

![image](https://github.com/user-attachments/assets/8736b60d-fc1e-4c92-a978-4f0a54cd122c)

I'm not sure Windows would ever have `adb.exe `in `%PATH%`?

We have existing logic for `$(AdbToolPath)` here:

https://github.com/dotnet/android/blob/d5dab91761a50eaa6f20f391792bf0dd20f5b801/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L765-L769

Because `_AndroidComputeRunArguments` depends on `Install`, `_ResolveMonoAndroidSdks` has already run at this point. This is similar to how other places using `$(AdbToolExe)` are working currently.

Reply:
Doh! I overlooked:

```xml
<RunCommand>$([System.IO.Path]::Combine ('$(AdbToolPath)', '$(RunCommand)'))</RunCommand>
```

which more or less does what I requested.